examples/upgrade/blue-green-route53/modules/eks_cluster/main.tf

```diff
@@ -0,0 +1,460 @@
+  ecsfrontend_route53_weight = var.ecsfrontend_route53_weight
+
+  tag_val_vpc            = var.vpc_tag_value == "" ? var.core_stack_name : var.vpc_tag_value
+  tag_val_private_subnet = var.vpc_tag_value == "" ? "${var.core_stack_name}-private-" : var.vpc_tag_value
+
+  node_group_name            = "managed-ondemand"
+  argocd_secret_manager_name = var.argocd_secret_manager_name_suffix
+
+  #---------------------------------------------------------------
+  # ARGOCD ADD-ON APPLICATION
+  #---------------------------------------------------------------
+
+  addon_application = {
+    path                = "chart"
+    repo_url            = var.addons_repo_url
+    ssh_key_secret_name = var.workload_repo_secret
+    add_on_application  = true
+  }
+
+  #---------------------------------------------------------------
+  # ARGOCD WORKLOAD APPLICATION
+  #---------------------------------------------------------------
+
+  workload_application = {
+    path                = var.workload_repo_path # <-- we could also to blue/green on the workload repo path like: envs/dev-blue / envs/dev-green
+    repo_url            = var.workload_repo_url
+    target_revision     = var.workload_repo_revision
+    ssh_key_secret_name = var.workload_repo_secret
+    add_on_application  = false
+    values = {
+      labels = {
+        env   = local.env
+        myapp = "myvalue"
+      }
+      spec = {
+        source = {
+          repoURL        = var.workload_repo_url
+          targetRevision = var.workload_repo_revision
+        }
+        blueprint                = "terraform"
+        clusterName              = local.name
+        karpenterInstanceProfile = "${local.name}-${local.node_group_name}"
+        env                      = local.env
+        ingress = {
+          type                  = "alb"
+          host                  = local.eks_cluster_domain
+          route53_weight        = local.route53_weight # <-- You can control the weight of the route53 weighted records between clusters
+          argocd_route53_weight = local.argocd_route53_weight
+        }
+      }
+    }
+  }
+
+  #---------------------------------------------------------------
+  # ARGOCD ECSDEMO APPLICATION
+  #---------------------------------------------------------------
+
+  ecsdemo_application = {
+    path                = "multi-repo/argo-app-of-apps/dev"
+    repo_url            = var.workload_repo_url
+    target_revision     = var.workload_repo_revision
+    ssh_key_secret_name = var.workload_repo_secret
+    add_on_application  = false
+    values = {
+      spec = {
+        blueprint                = "terraform"
+        clusterName              = local.name
+        karpenterInstanceProfile = "${local.name}-${local.node_group_name}"
+
+        apps = {
+          ecsdemoFrontend = {
+            #repoURL        = "https://github.com/aws-containers/ecsdemo-frontend"
+            repoURL        = "https://github.com/allamand/ecsdemo-frontend"
```

Comment:
Does this need to be aws-containers repo?

Reply:
I have a PR ongoing, but let treat this appart, i put back to aws-containers now